user_docs/en/userGuide.t2t

```diff
@@ -346,6 +346,10 @@ When within a table, the following key commands are also available:
 | Move to last column | control+alt+end | Moves the system caret to the last column (staying in the same row) |
 | Move to first row | control+alt+pageUp | Moves the system caret to the first row (staying in the same column) |
 | Move to last row | control+alt+pageDown | Moves the system caret to the last row (staying in the same column) |
+| Speak column downwards | ``NVDA+control+alt+downArrow`` | Reads column vertically from the current table cell downwards to the last cell in the column. |
+| Speak row rightwards | ``NVDA+control+alt+rightArrow`` | Reads row horizontally from current table cell rightwards to the last cell in the row. |
+| Speak entire column | ``NVDA+control+alt+upArrow`` | Reads current column vertically from top to bottom without moving the system caret. |
+| Speak entire row | ``NVDA+control+alt+leftArrow`` | Reads current row vertically from left to right without moving the system caret. |

```

Comment:
Another comment forgotten in the previous review:
* Replace "Speak entire column/row" with "Read entire column/row", to make something similar as "Read current line" and "Read current selection" in the table just above. On the contrary "Speak" in the names of commands are used only for "Speak typed characters/typed words/command keys" which are other types of commands.
* "Read current column/row" -> "Read the current column/row":added "the" to avoid telegraphic style.

```suggestion
| Read entire column | ``NVDA+control+alt+upArrow`` | Reads the current column vertically from top to bottom without moving the system caret. |
| Read entire row | ``NVDA+control+alt+leftArrow`` | Reads the current row vertically from left to right without moving the system caret. |
```